pkg/wrangler/context.go

```diff
@@ -262,10 +290,31 @@ func (w *Context) WithAgent(userAgent string) *Context {
 	wContextCopy.API = wContextCopy.api.WithAgent(userAgent).V1()
 	wContextCopy.CRD = wContextCopy.crd.WithAgent(userAgent).V1()
 	wContextCopy.Plan = wContextCopy.plan.WithAgent(userAgent).V1()
+	if w.DeferredCAPIRegistration.CAPIInitialized {
+		wContextCopy.CAPI = wContextCopy.capi.WithAgent(userAgent).V1beta1()
+	} else {
+		go wContextCopy.BackfillCAPI(&wContextCopy)
```

Comment:
:warning: I believe that this is a bug. It seems to attempt to backfill `wContextCopy` from itself. I do not believe that this can work.
```suggestion
		go wContextCopy.BackfillCAPI(w)
```